ludwig/trainers/trainer.py

```diff
@@ -1236,6 +1240,25 @@ def set_steps_to_1_or_quit(self, signum, frame):
                 signal.signal(signal.SIGINT, self.original_sigint_handler)
             sys.exit(1)
 
+    def resume_files_exist(
+        self,
+        training_progress_tracker_path: str,
+        training_checkpoint_path: str,
+    ) -> bool:
+        missing_files = []
+        # training_progress.json
+        if not path_exists(training_progress_tracker_path):
```

Comment:
Is there a reason why this doesn't need to be is_coordinator() guarded?

Reply:
Fixed! It should be guarded